composer.json

```diff
@@ -8,6 +8,7 @@
     "license": "Apache-2.0",
     "require": {
         "php": "^7.4 || ^8.0",
+        "cloudevents/sdk-php": "^v1.0.1",
```

Comment:
This needs to be added to the API package's composer config as well.
(We should have a test which checks for this, for example running the unit tests of a package with its composer file.)

Reply:
done